fcrepo-import/src/main/java/org/fcrepo/importer/ArgParser.java

```diff
@@ -51,6 +58,43 @@ public ArgParser() {
         importExportOption.setArgs(1);
         importExportOption.setArgName("mode");
         cmdOptions.addOption(importExportOption);
+
+        // Resource option
+        final Option resourceOption = new Option("r", "resource", true, "Resource (URI) to import/export");
+        resourceOption.setRequired(true);
+        resourceOption.setArgs(1);
+        resourceOption.setArgName("resource");
+        cmdOptions.addOption(resourceOption);
+
+        // Binary Directory option
+        final Option binDirOption = new Option("b", "binDir", true, "Directory to store binaries (files)");
+        binDirOption.setRequired(false);
+        binDirOption.setArgs(1);
+        binDirOption.setArgName("binDir");
+        cmdOptions.addOption(binDirOption);
+
+        // Description Directory option
+        final Option descDirOption = new Option("d", "descDir", true, "Directory to store RDF descriptions");
```

Comment:
`Directory to store RDF descriptions`... should this instead be `Directory of stored RDF descriptions`?